deckhouse-controller/crds/crd.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crds
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"strings"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"sigs.k8s.io/yaml"
+)
+
+//go:embed *.yaml
+var FS embed.FS
```

Comment:
what for?
We have a one-time read on startup. Don't think we have to keep this in a memory. It's not often-used file.
Also, sometimes it's comfortable to change a CRD in a cluster and reload the d8